regression/contracts/trivial_contract_enforce/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^\[main.assertion.\d+\] line \d+ assertion foo\(\&n\) != 15\: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+Check whether CBMC doesn't crash when enforcing trivial contracts on functions.
```

Comment:
I am not really sure what "trivial" means in this context.  Checking whether a pointer is `NULL` is not always trivial.

Reply:
I updated the test description. The "trivial" refers to contracts with only a requires clause, which would be basically adding an assumption. It's not related to nullness check.